General/Glossary.md

```diff
@@ -58,14 +84,25 @@ Some methods holding this primitive pragma also contain source code written in S
 In Pharo, each method belongs to a protocol. A protocol has a name and allows one to group methods that are related together. For example, the protocol `accessing` is widely used in the system to hold accessor and mutator methods.
 
 ## Receiver
-The receiver of a message send is the instance of a class implementing the method corresponding to the message sent receiving this message.
+The receiver of a [message](#message) send is the instance of a class implementing the method corresponding to the message sent receiving this message.
 
 ## Selector
 A selector corresponds to the signature of a method in the system. Since Pharo is not statically typed, the selector of a method is simply a Symbol. On a CompiledMethod, it is possible to send `#selector` message to retrieve this Symbol.
 
 ## Self
+Self is the [receiver](#receiver) of a [message](#message) an object send to itself. This means that the [lookup](#lookup) of a method sent will start by the sender of this message then, if not found, it will proceed with its superclass.
```

Comment:
```suggestion
Self is the [receiver](#receiver) of a [message](#message) an object sends to itself. This means that the [lookup](#lookup) of a method send will start by the sender of this message then, if not found, it will proceed with its superclass.
```